src/tests/RoomInfo.test.jsx

```diff
@@ -0,0 +1,24 @@
+import { test, describe, expect } from 'vitest';
```

Comment:
- [ ] Although your tests are on point but as the requirement says, you must use React-testing library for integration tests.
- Create integration tests for your application using the React Testing Library.
- You may need to mock the access to the API, so that your tests don't send actual requests.
- You may need to mock the connection to the Redux Store.

Reply:
@Zuhaib042  I misunderstood the requirements about the testing part, thank you so much for giving you feedback on that. Now I can fix that according to the requirements 💯